### PR TITLE
[1LP][RFR] Adding stack bundle retire test

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run
{{pytest: cfme/tests/services/test_provision_stack.py::test_retire_catalog_bundle_service_orchestration_item --long-running   --use-provider ec2west -vvvv }}

It fails on 5.11. we have open bz: 1754543

### CODE REVIEW

File: cfme/tests/services/test_provision_stack.py [hunk 1]
```diff
@@ -395,3 +395,82 @@ def test_stack_template_azure():
...
+            2. Provisioned the catalog bundle with ServiceOrchestration item
+            3. Navigate to My service page
+            4. Retired the bundle
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Catalog bundle should retire with no error
+
+    """
```
Comment (ganeshhubale) [inline cfme/tests/services/test_provision_stack.py]: remove blank line

File: cfme/tests/services/test_provision_stack.py [hunk 2]
```diff
@@ -395,3 +395,82 @@ def test_stack_template_azure():
...
+@pytest.mark.meta(automates=[1684092])
+@pytest.mark.meta(blockers=[BZ(1754543)])
+@pytest.mark.customer_scenario
+@pytest.mark.tier(2)
+@pytest.mark.provider([EC2Provider], selector=ONE, scope='module')
+def test_retire_catalog_bundle_service_orchestration_item(appliance, request, catalog_item,
+                                                          stack_data):
+    """
+
+    Bugzilla:
```
Comment (ganeshhubale) [inline cfme/tests/services/test_provision_stack.py]: remove blank line

File: cfme/tests/services/test_provision_stack.py [hunk 3]
```diff
@@ -395,3 +395,82 @@ def test_stack_template_azure():
         upstream: yes
     """
     pass
+
+
+@pytest.mark.meta(automates=[1684092])
+@pytest.mark.meta(blockers=[BZ(1754543)])
+@pytest.mark.customer_scenario
```
Comment (ganeshhubale) [inline cfme/tests/services/test_provision_stack.py]: Add both In oneline.
```

@pytest.mark.meta(automates=[1684092])
@pytest.mark.meta(blockers=[BZ(1754543)])```

File: cfme/tests/services/test_provision_stack.py [hunk 4]
```diff
@@ -395,3 +395,78 @@ def test_stack_template_azure():
...
+    # Ordering service catalog bundle
+    service_catalogs = ServiceCatalogs(
+        appliance, catalog_bundle.catalog, catalog_bundle.name, stack_data)
+
+    service_catalogs.order()
+    request_description = (
+        f'Provisioning Service [{catalog_bundle.name}] from [{catalog_bundle.name}]'
+    )
+
+    provision_request = appliance.collections.requests.instantiate(request_description)
```
Comment (digitronik) [inline cfme/tests/services/test_provision_stack.py]: I'm suspect why you need to instantiate this provisioning request? As per my info `order` method itself returning this.

Comment (niyazRedhat) [reply cfme/tests/services/test_provision_stack.py]: @digitronik to get the last message(provisioned service name)  from the service request table. 
 
Last Message
`[EVM] Service [cat_bundle_z69K-20200325-025939] Provisioned Successfully`

Comment (niyazRedhat) [reply cfme/tests/services/test_provision_stack.py]: Made changes as per your suggestion

**Pull request merged**